liblumen_alloc/src/erts/process/alloc/heap_alloc.rs

```diff
@@ -541,6 +531,64 @@ pub trait HeapAlloc {
         // Return box to tuple
         Ok(Term::make_boxed(tuple_ptr))
     }
+
+    /// Constructs a `Closure` from a slice of `Term`
+    ///
+    /// Be aware that this does not allocate non-immediate terms in `elements` on the process heap,
+    /// it is expected that the slice provided is constructed from either immediate terms, or
+    /// terms which were returned from other constructor functions, e.g. `make_binary_from_str`.
```

Comment:
`make_binary_from_str` is called `binary_from_str`.  The tuple comments have the old name.